### PR TITLE
🌐 Add Japanese translation for Deployment - Manually

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates deployment/manually.md in Japanese.

To all reviewers. This section is only separated from #1948 (with few changes), but please review it.

### CODE REVIEW

**Review by komtaki — APPROVED** [review]
Thank you for the tough work.
LGTM 👍

File: docs/ja/docs/deployment/manually.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    <div class="termy">
+
+    ```console
+    $ pip install uvicorn
+
+    ---> 100%
+    ```
+
+    </div>
```
Comment (delhi09) [inline docs/ja/docs/deployment/manually.md:19]: It seems the translation of the following sentences is missing.

> Tip
> By adding the standard, Uvicorn will install and use some recommended extra dependencies.
> That including uvloop, the high-performance drop-in replacement for asyncio, that provides the big concurrency performance boost.

**Review by delhi09 — COMMENTED** [review]
It looks mostly good.
But the translation of the one "Tip" section seems to be missing.

**Comment by tokusumi** [comment]
@delhi09 Thank you for review!
I updated translation. please check it again.

**Review by delhi09 — APPROVED** [review]
LGTM！

**Comment by tiangolo** [comment]
Thank you very much for updating all this @tokusumi :bow: :cake: 

And as always, thanks @komtaki and @delhi09 for the reviews :mag: :coffee:

**Pull request merged**